x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -4335,11 +4335,6 @@
     "xpack.apm.serviceDetails.metricsTabLabel": "指标",
     "xpack.apm.serviceDetails.nodesTabLabel": "JVM",
     "xpack.apm.serviceDetails.transactionsTabLabel": "事务",
-    "xpack.apm.serviceMap.avgCpuUsagePopoverMetric": "CPU 使用（平均）",
-    "xpack.apm.serviceMap.avgErrorsPerMinutePopoverMetric": "每分钟错误数（平均）",
-    "xpack.apm.serviceMap.avgMemoryUsagePopoverMetric": "内存使用（平均）",
-    "xpack.apm.serviceMap.avgReqPerMinutePopoverMetric": "每分钟请求数（平均）",
-    "xpack.apm.serviceMap.avgTransDurationPopoverMetric": "事务持续时间（平均）",
```

Comment:
I think only the translation for errors would have changed, so the others should only update their ids